Library/Homebrew/formulary.rb

```diff
@@ -118,6 +121,7 @@ def initialize(alias_path)
       path = alias_path.resolved_path
       name = path.basename(".rb").to_s
       super name, path
+      @install_name = alias_path.basename.to_s
```

Comment:
How about we go for `alias_path`, and make it return `nil` rather than the formula path if an alias wasn't used?

Then we could use `formula.alias_path || formula.path` when passing a reference to it, and the meaning would be clearer all round.